Source/WebKit/UIProcess/WebPageProxy.cpp

```diff
@@ -1007,7 +1007,7 @@ bool WebPageProxy::hasRunningProcess() const
 
 void WebPageProxy::notifyProcessPoolToPrewarm()
 {
-    Ref processPool = m_legacyMainFrameProcess->processPool();
+    Ref processPool = protectedLegacyMainFrameProcess()->processPool();
```

Comment:
It's pretty silly that we have to use the protected variant just so that we can call `processPool()`.
Maybe we can inline the definition of `WebProcessProxy::processPool()` into `WebProcessPool.h`.
`WebProcessPool.h` already includes `WebProcessProxy.h` and any code that calls `WebProcessProxy::processPool()` must include both `WebProcessProxy.h` and `WebProcessPool.h` so it's not that unnatural.

Reply:
Oh cool, that's a neat way to fix it. I didn't realize that the function definition could be put in a different header that way. Fixed! Let me know if it looks good now.